server/Entities/Migrations/20200529192154_AddUserOauthField.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Entities.Migrations
+{
+    public partial class AddUserOauthField : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<bool>(
+                name: "IsOauth",
```

Comment:
do we really need this field?

Reply:
you can extract google_id (or smth alike) from google response and store it instead of "IsOauth"